EpiserverRedirects.Tests/Model/UrlPathTests.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Forte.EpiserverRedirects.Model;
+using Xunit;
+
+namespace Forte.EpiserverRedirects.Tests.Model
+{
+    public class UrlPathTests
+    {
+        [InlineData("/some/path/", "/some/path")]
+        [InlineData("some/path/", "/some/path")]
+        [InlineData("some/path", "/some/path")]
+        [InlineData("", "")]
+        [InlineData(null, "")]
```

Comment:
Just a small suggestion to include whitespace-only string as an additional test case